src/installer/hash.ts

```diff
@@ -152,20 +159,24 @@ function parseHashFile(hashFileContents: string, fileToHash: string): string {
  * Since yq has checksum file in different format from other clients.
  * @returns The hash for fileToHash, as extracted from the hashFileContents.
  */
-function parseHashFileForYq(hashFileContents: string, fileToHash: string): string {
+function parseHashFileForYq(hashFileContents: string, checksumHashesOrdercontents: string, fileToHash: string): string {
+
+    const checksumHashesOrder = checksumHashesOrdercontents.split("\n");
```

Comment:
can you link to an existing release's hash file and orders file here so another developer can easily find the file you are trying to parse

Reply:
Okay :+1: